basyx.aasservice/basyx.aasservice-core/src/test/java/org/eclipse/digitaltwin/basyx/aasservice/AasServiceSuite.java

```diff
@@ -94,8 +99,8 @@ public void addSubmodelReference() {
 	@Test
 	public void removeSubmodelReference() {
 		AssetAdministrationShell expected = DummyAssetAdministrationShellFactory.create();
-		AasService aasService = getAasService(expected);
 		DummyAssetAdministrationShellFactory.addDummySubmodelReference(expected);
+		AasService aasService = getAasService(expected);
```

Comment:
Just for understanding: why have you moved this line down?

Reply:
Because otherwise the test would fail, as the SubmodelReference is not added to the AAS.
The SubmodelReference needs to be added before the AAS is registered in the AASService.
I don't know why it worked before, but it didn't after my changes